### PR TITLE
refactor(PPDSC-2606): update feature card component & copy

### PR DESCRIPTION
**Pull request by JohnTParsons**
PPDSC-2606

**What**

1. Background - why this is needed
Update the feature card component, and copy updated on the relevant Theme, and Patterns section pages on the site, as requested by design

2. What did you do
Updated the copy and refactored FeatureCard to use StandaloneLink

3. What does the reviewers should expect
Should match Figma specs

**I have done:**
- [x] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [x] Updated relevant documentation

**I have tested manually:**
- [x] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [x] The screen reader reads and flows through the elements as expected.
- [x] There are no new errors in the browser console coming from this PR.
- [x] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

Used to display a button with an arrow in it

**After:**

Now displays a link followed by an arrow
PPDSC-2606-after

**Who should review this PR:**

**How to test:**

To test go to the Docs home page, where two feature cards can be viewed.

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2606-refactor-feature-card
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2606-refactor-feature-card/storybook

File: src/link/style-presets.ts [hunk 1]
```diff
@@ -38,6 +38,24 @@ export default {
     },
     'focus-visible': defaultFocusVisible,
   },
+  linkStandaloneInversePersistent: {
+    base: {
+      color: '{{colors.inkLight010}}',
+      iconColor: '{{colors.inkLight010}}',
+      textDecoration: 'none',
+    },
```
Comment (mutebg) [inline src/link/style-presets.ts:46]: Do we need to have this stylePreset in newskit or can be only for the website?

File: site/components/feature-card/feature-card.tsx [hunk 2]
```diff
@@ -75,32 +96,42 @@ const FeatureCardHorizontal = React.forwardRef<
...
     const theme = useTheme();
     const titleTypographyPreset = getToken(
       {theme, overrides},
       '',
       'title',
       'typographyPreset',
     );
+    /* const titleStylePreset = stylePrefix
+      ? {stylePreset: `${stylePrefix}ContainerNonInteractive`}
+      : {stylePreset: 'inkWhiteContrast'}; */
```
Comment (mutebg) [inline site/components/feature-card/feature-card.tsx:122]: Some leftovers here

File: site/components/feature-card/feature-card.tsx [hunk 3]
```diff
@@ -48,25 +47,47 @@ const OptionalLinkWrapper: React.FC<OptionalLinkWrapperProps> = ({
...
+        >
+          <Stack flow="horizontal-center" as="span">
+            <TextBlock typographyPreset="utilityButton020" as="span">
+              {label}
+            </TextBlock>
+            {icon || (
+              <IconFilledChevronRight overrides={{size: 'iconSize020'}} />
+            )}
+          </Stack>
+        </TextBlock>
```
Comment (mutebg) [inline site/components/feature-card/feature-card.tsx:87]: We have lots of nested elements here, 
wondering if we can simplify that using `<LinkStandalone as="span">`

File: site/components/feature-card/feature-card.tsx (hunk 2 above)
Comment (JohnTParsons) [reply site/components/feature-card/feature-card.tsx]: now removed

File: src/link/style-presets.ts (hunk 1 above)
Comment (mutebg) [reply src/link/style-presets.ts:46]: `site/theme/style-presets.ts` here is only site-related style presets

File: site/components/feature-card/feature-card.tsx (hunk 3 above)
Comment (JohnTParsons) [reply site/components/feature-card/feature-card.tsx:87]: Lines 64 to 71 are for rendering <LinkStandalone> but lines 74 to 87 (as highlighted above) are for displaying text followed by an arrow on a card that has `href` prop set, meaning the whole card is interactive. <LinkStandalone> cannot be used in that context or Next won't allow <a> to be embedded inside an <a>

Comment (Xin00163) [reply site/components/feature-card/feature-card.tsx:87]: Would using a flag work?
```
       <Flag
          overrides={{
            typographyPreset: 'utilityButton020',
            stylePreset: combinedOverrides?.stylePreset,
          }}
        >
          {label}
          {icon || <IconFilledChevronRight overrides={{size: 'iconSize020'}} />}
        </Flag>
```

Comment (JohnTParsons) [reply site/components/feature-card/feature-card.tsx:87]: Will try <LinkStandalone as="span" ...> for the not href use case...

**Review by mutebg — APPROVED** [review]

**Review by Vanals — APPROVED** [review]

**Pull request merged**